stock_available_portal/readme/USAGE.rst

```diff
@@ -0,0 +1 @@
+* Open **Settings** > **Inventory**. In the section **Stock available to promise** set domain for the access on "My Products"  in portal for users and set domain for the showing sets product on portal.
```

Comment:
Open **Settings** > **Inventory**.
 In  **Stock available to promise** section there are domains for products and users. 
Product domain defines which products will be visible in portal.
User domain defines which users have access to products in portal.

Reply:
Resolved